tests/realtikvtest/addindextest3/ingest_test.go

```diff
@@ -448,6 +448,27 @@ func TestAddIndexMockFlushError(t *testing.T) {
 	require.True(t, strings.Contains(jobTp, "ingest"), jobTp)
 }
 
+func TestAddIndexDiskQuotaTS(t *testing.T) {
```

Comment:
I know add-index with DXF shouldn't have such issue, but can you add a case to cover it too?

Reply:
OK, I can add one test for DXF enabled. But we should leave more test improvements in future in a separate PR. Like iterate all variables combinations like {DXF = on/off, disk quota = on/off, ...} and provide a helper function to prepare environment, and run the testing logic will all combinations.